src/main/java/bio/terra/service/filedata/FileMetadataUtils.java

```diff
@@ -22,6 +22,8 @@ public class FileMetadataUtils {
 
   public FileMetadataUtils() {}
 
+  // TODO: this currently returns the directory as "" if you pass in a one-level deep item
+  // https://broadworkbench.atlassian.net/browse/DR-3005 is to fix (changing breaks tests badly)
```

Comment:
Bummer that this drive-by change didn't work, but I appreciate that you left a trail of breadcrumbs with this comment!